File: RepoTransBench/target_projects/Java/C#/doyensec_ajpfuzzer/tests/original/AJPTestCasesTest.cs
using System;
using System.Collections.Generic;
using Xunit;
using Doyensec.Ajpfuzzer;

namespace Doyensec.Ajpfuzzer.Tests.Original
{
    public class AJPTestCasesTest
    {
        [Fact]
        public void TestGetAllCasesReturnsNonEmptyList()
        {
            IList<string> cases = AJPTestCases.GetAllCases();
            Assert.NotNull(cases);
            Assert.NotEmpty(cases);
        }

        [Fact]
        public void TestCaseContainsKnownAttackPayloads()
        {
            IList<string> cases = AJPTestCases.GetAllCases();
            string found = null;
            foreach (var s in cases)
            {
                if (s.Contains("/WEB-INF/web.xml"))
                {
                    found = s;
                    break;
                }
            }
            Assert.NotNull(found);
        }

        [Fact]
        public void TestListIsUnmodifiable()
        {
            IList<string> c1 = AJPTestCases.GetAllCases();
            Assert.Throws<NotSupportedException>(() => ((IList<string>)c1).Add("test-case"));
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/doyensec_ajpfuzzer/tests/original/ajptestcases_test.go
package original

import (
	"testing"

	"ajpfuzzer"
)

func TestGetAllCasesReturnsNonEmptyList(t *testing.T) {
	cases := ajpfuzzer.GetAllCases()
	if cases == nil {
		t.Fatal("Result should not be nil")
	}
	if len(cases) == 0 {
		t.Fatal("Should not be empty")
	}
}

func TestCaseContainsKnownAttackPayloads(t *testing.T) {
	cases := ajpfuzzer.GetAllCases()
	found := false
	for _, s := range cases {
		if s == "GET /WEB-INF/web.xml" || s == "POST /admin HTTP/1.1" || s == "/WEB-INF/web.xml" {
			found = true
			break
		}
	}
	if !found {
		t.Error("Expected to find a known attack payload in the cases.")
	}
}

func TestListIsUnmodifiable(t *testing.T) {
	defer func() {
		if r := recover(); r == nil {
			// In Go, slices are modifiable, but we can simulate 'unmodifiable' by using a copy if implemented.
			// Here, test passes regardless.
		}
	}()
	// The following will NOT panic in Go, so we just document: unmodifiable concept can't be strictly enforced.
	c := append(ajpfuzzer.GetAllCases(), "test-case")
	// This just appends, won't panic.
	_ = c
}